recipes/ezomero/meta.yaml

```
@@ -0,0 +1,34 @@
+{% set version = "2.1.0" %}
+{% set name = "ezomero" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 729db284c27067b8accb127c6501b2205f63e639045339e6939f7c991f0759c7
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  run_exports:
+    - {{ pin_subpackage("ezomero", max_pin="x.x") }}
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - omero-py =5.16
```

```suggestion
    - python >=3.8
    - omero-py =5.16
```
Could you please add python to the list of `run` dependencies as  the package is labeled `noarch: python`?